views/news/news-item.js

```diff
@@ -27,6 +27,10 @@ function NewsItemContents({content}: StoryType) {
       img {
         max-width: 100%;
         height: auto;
+        display: block;
+        margin: auto;
+        margin-top: 12px;
+        margin-bottom: 12px;
```

Comment:
Could you use a shorthand `margin` prop here? I'm thinking `margin: 12px auto;` instead of the three declarations. 

That way we don't override our initial declaration.